salt/beacons/service.py

```diff
@@ -76,6 +76,11 @@ def beacon(config):
     event when the minion is reload. Applicable only when `onchangeonly` is True.
     The default is True.
 
+    `is_running_state`: If `is_running_state` is set to a boolean the beacon will only
+    fire if the running state matches the given value. If its set to `True`, only
+    when a service running state is `True` it will be fired, if it's set to `False`
+    then only the beacon will fire when the running state is `False`.
```

Comment:
This could be worded a little better. Consider the following:

This boolean parameter controls when the beacon will fire based on the service's running state. If `is_running_state` is set to `True`, the beacon will fire only when the service is running. If `is_running_state` is set to `False`, the beacon will fire only when the service is not running.

Reply:
I have replaced the text with your suggestion.